src/keystone/keystone_api.h

```diff
@@ -13,15 +13,15 @@
 // limitations under the License.
 
 #include <iostream>
-#include <sys/types.h>
-#include <sys/stat.h>
-#include <stdint.h>
+//#include <sys/types.h>
```

Comment:
nit: Can we deleted these if they are not used?

Reply:
@jlmucb these were in the `.h` file you sent over -- are these needed for the certifier? Can uncomment or delete them depending on that.